addons/website_quote/data/website_quote_demo.xml

```diff
@@ -317,4 +317,288 @@
             <field name="price_unit">9000.00</field>
             <field name="discount">10</field>
         </record>
+
+        <record id="website_quote_template_section" model="sale.quote.template">
```

Comment:
Same that before. Couldn't you reuse an existing demo data instead of creating a new one? The website_description seems to be copy/pasted. This adds a lot of source terms to translate (for nothing).